### ISSUE TITLE
[BUG] Documentation Error. Minor Fix.

### ISSUE THREAD

**Issue by schrodyn** [vertexproject/synapse#1301]
**Documentation Typo**
On the following page there is a documentation mismatch:

https://vertexprojectsynapse.readthedocs.io/en/latest/synapse/autodocs/datamodel_forms.html

Under the form section for `it:app:snort:hit` is writes:

```
Properties:

  :rule / it:app:snort:hit:rule
    The yara rule that matched the file.
```

This should be `snort rule` and not `yara rule`.

**Comment by jnwatson** [comment]
Thanks!

**Comment by vEpiphyte** [comment]
This is fixed as collateral in #1304
